bin/pycbc_live

```diff
@@ -215,6 +215,105 @@ class LiveEventManager(object):
                     triggers['foreground/ifar'], pvalue, self.pvalue_livetime)
             triggers['foreground/pvalue_{}'.format(ifo)] = pvalue
 
+
+    def setup_optimize_snr(self, results, ifos, bank, fname, data_readers, gid):
+        template_id = \
+            results['foreground/{}/template_id'.format(ifos[0])]
+        p = props(bank.table[template_id])
+        apr = p.pop('approximant')
+        min_buffer = bank.minimum_buffer + 0.5
+        buff_size = \
```

Comment:
@titodalcanton @GarethCabournDavies So I think this should pass 

```
bank.approximant(template_id)
```

to the function and then it should work fine.